### PR TITLE
chore: fixing postman API publishing

### LINKED ISSUE #1949: Update Postman API publish on tag
## Context
The Postman Git integration does not recognize tags as branches, and as a result, [fails to publish](https://github.com/interledger/rafiki/actions/runs/6259425191/job/16995418020) the API in the Github Action.

Instead of publishing the API version from the ref being the tag, we can just publish from `main` when we make a new Rafiki release.

### PR DESCRIPTION
**Pull request by mkurapov**
- Reverting postman/collections/Interledger.json file to alpha5 version and re-adds all of the newest changes since then to properly making publishing Postman APIs work (tested publishing manually and via the CLI)

## Changes proposed in this pull request

Ever since [commit](https://github.com/interledger/rafiki/commit/2023d293e3ee5c5945b8fa374cbaaf6526d0b64d), the Postman API publish to `main` has not been working, since in the PR, the collection file, Interledger.json was exported and updated manually instead of via the Postman Git integration. It looks like each request needs an `id` to properly publish the API. This PR takes the collection from the working API collection of the alpha 5 tag, ands adds the changes since the alpha 5 release to make Postman API publishing work again.

## Context

Fixes #1949 

Also opened up an issue on the Postman side: https://github.com/postmanlabs/postman-app-support/issues/12577 to see if I can properly debug if this happens in the future.

## Checklist

- [x] Related issues linked using `fixes #number`
- [ ] Tests added/updated
- [ ] Documentation added
- [ ] Make sure that all checks pass
- [x] Postman collection updated

### CODE REVIEW

File: postman/collections/Interledger.json [hunk 1]
```diff
@@ -1145,49 +1343,19 @@
 						}
 					},
 					"response": []
-				},
-				{
-					"name": "Get Receiver (remote incoming payment) Copy",
```
Comment (mkurapov) [inline postman/collections/Interledger.json:1150]: Added here: https://github.com/interledger/rafiki/pull/2332/files#diff-0ded81fc849a6b9aa9f8ce22cc3349b819ec40bdae363e87a37af9c5f801c6c4R908

File: postman/collections/Interledger.json [hunk 2]
```diff
@@ -995,32 +1205,8 @@
 					"response": []
 				},
 				{
-					"name": "Withdraw Event Liquidity",
```
Comment (mkurapov) [inline postman/collections/Interledger.json:998]: still here, just moved: https://github.com/interledger/rafiki/pull/2332/files#diff-0ded81fc849a6b9aa9f8ce22cc3349b819ec40bdae363e87a37af9c5f801c6c4R1236

**Comment by sabineschaller** [comment]
Should we just merge and hope for the best or is there a way to actually test this?

**Comment by mkurapov** [comment]
@sabineschaller 
I ran the Postman CLI `publish api ...` command locally (identical to the Github Actions one) and it worked

**Review by BlairCurrey — APPROVED** [review]
checked out locally. looks good.

**Pull request merged**